lib/runner.js

```diff
@@ -247,6 +252,19 @@ internals.enableSkip = (element) => {
     element.options.skip = true;
 };
 
+internals.markParentsAsOnly = (element) => {
+
+    if (!element.options.skip) {
```

Comment:
Should we assert in this case and inform the developer that they must either remove the `skip` or `only` in the experiment tree?

Reply:
I think there is a use case where the dev would do this on purpose. If multiple tests are marked as `only`, but for a given test run they want to skip some of them. Skipping a common parent experiment can be a lot quicker than deleting potentially several `only`s. This can be pretty handy when you are working with many levels of experiments.

Having said that, I can also see potential for some confusion as to why a test marked with `only` isn't running if the dev doesn't notice that it was skipped higher up. I think that case would end up being similar to the older approach of throwing when there are multiple `only`s. 

Maybe a section in the docs explaining the logic?